minetest/minetest
# Add 4 missing fallback/default textures

This PR adds the missing default/fallback textures to the engine:

* `heart.png`
* `bubble.png`
* `crack_anylength.png`
* `wieldhand.png`

The first 3 are just pulled from minimal.

The final one is just a green rectangle to match the color of our current `player.png`/`player_back.png`. I believe these files are deliberately ugly to force modders to replace them?

I claim no credit to any of these files. So I assume there's a default license. Right?

## Justification

This is basically the logical continuation of https://github.com/minetest/minetest/commit/b52a79076d2e209686fca678fc574fab71b67359

## How to test

* Create an empty game, add some test nodes: A node that is diggable, a node in which you drown
* Generate a simple world with these test nodes
* Start the game
* Look at your hearts and wieldhand
* Dig the diggable test node and check the crack texture
* Go into the drowning node and check your bubbles

## Caveat

It appears the `wieldhand.png` textures is not used by the engine default. When I look at my hand, I see a missing texture.

This appears to be a bug / missing feature in Minetest. However, `wieldhand.png` is mentioned in docs.

However, this apparent bug should not block this PR, as the bugfix would be just to properly use the `wieldhand.png`.
